src/main/java/com/basesyntax/strategy/impl/StrategyImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.basesyntax.strategy.impl;
+
+import com.basesyntax.dao.OperationHandler;
+import com.basesyntax.dao.impl.OperationHandlerBalanceImpl;
+import com.basesyntax.dao.impl.OperationHandlerPurchaseImpl;
+import com.basesyntax.dao.impl.OperationHandlerReturnImpl;
+import com.basesyntax.dao.impl.OperationHandlerSupplyImpl;
+import com.basesyntax.enums.Operation;
+import com.basesyntax.strategy.Strategy;
+import java.util.HashMap;
+import java.util.Map;
+
+public class StrategyImpl implements Strategy {
+    private Map<String, OperationHandler> strategyMap = new HashMap<>();
+
+    @Override
+    public void addStrategyType(String operationType, OperationHandler operationHandler) {
+        strategyMap.put(Operation.BALANCE.getOperation(), new OperationHandlerBalanceImpl());
```

Comment:
Method parameters are not used, but should be (and method is not working as intended, because ignoring parameters while adding to map hardcodedly)